handout/images/README.md

```diff
@@ -0,0 +1,16 @@
+# Image Attributions
+
+- advanced-component.jpg - https://commons.wikimedia.org/wiki/File:GB_Network_PCI_Card.jpg
+- angular-di.jpg *UNCERTAIN* Cannot Find Image
+- angular1-vs-angular2.jpg - *UNCERTAIN* Ask David
+- angular2-cover2.jpg - Chris P made this?
+- app-screenshot.png - Screenshot of our plunker
+- change-detection.jpg -  https://pixabay.com/en/coins-handful-russia-ruble-kopek-650779/ - cli-folder-setup.png - Screenshot of files
+- di-tree.png - Michael made this
+- es6-typescript.png - Michael made this
+- flat-map.png - Observable docs? Open source?
+- migration.jpg - https://pixabay.com/en/great-migration-africa-animal-1021460/
+- pipes.jpg - https://pixabay.com/en/pipe-plumbing-connection-pipeline-406906/
+- simple-output.png - Screenshot of our output
+- tools.jpg - http://commons.wikimedia.org/wiki/File:Hand_tools.jpg
+- uluru.jpg - https://commons.wikimedia.org/wiki/File:Ayers_Rock_Uluru.jpg
```

Comment:
@bennett000 for stuff that we made internally/you made, should we license them with CC with similar captions?